command.go

```diff
@@ -751,7 +755,12 @@ func (c *Command) Runnable() bool {
 
 // Determine if the command has children commands
 func (c *Command) HasSubCommands() bool {
-	return len(c.commands) > 0
+	for _, sub := range c.commands {
+		if sub.Runnable() {
+			return true
+		}
+	}
+	return false
 }
```

Comment:
Sorry, I wasn't aware of `HasRunnableSubCommands()`. Might have happened when I first created this PR. Reverted to the original behavior.